wb_interpolate.py

```diff
@@ -0,0 +1,83 @@
+"""
+input: points in form (x,y)
+output: coefficients of interpolated polynomial
+"""
+
+#hardcode the input
+
+
+# an encoder and decoder for Reed-Solomon codes with coefficients in Z/p for a prime p
+# decoder uses the Berlekamp-Welch algorithm
+
+# for solving a linear system
+from linearsolver import someSolution
+
+from field import GF
+from polynomial import polynomialsOver
+
+# n is the total number of messages send out == total number of nodes
+# k is the total number of shared secret, k = t + 1 with degree t polinomial 
```

Comment:
Slight terminology correct - this is Reed Solomon not secret share encoding. So `k` is not the number of shared secrets, but it could be called the number of code symbols.
Polinomial -> Polynomial

Reply:
Fixed!